frontend/src/components/CodeSmellList.tsx

```diff
@@ -0,0 +1,40 @@
+import { Badge, Flex } from '@chakra-ui/react';
+
+// const CodeSmells = [
+//   'Comment',
+//   'Incorrect typing',
+//   'Typo',
+//   'Logic Error',
+//   'Invocation Error'
+// ];
```

Comment:
Unnecessary comment, either delete it or use it with zip function to generate the mappings below